packages/orbit-components/src/Inline/Inline.stories.js

```diff
@@ -31,7 +31,7 @@ storiesOf("Inline", module)
   .add(
     "Default",
     () => {
-      const spacing = select("spacing", SPACING, SPACING[1]);
+      const spacing = select("spacing", Object.values(SPACING), SPACING[1]);
```

Comment:
`SPACING[1]` is `undefined`, need to use e.g. `SPACING.XXXSMALL`. It applies to all other same assertions in this file.